src/python/pants/backend/python/target_types.py

```diff
@@ -262,9 +280,9 @@ class PexBinary(Target):
     alias = "pex_binary"
     core_fields = (
         *COMMON_TARGET_FIELDS,
-        InterpreterConstraintsField,
         OutputPathField,
-        PexBinarySources,
+        DeprecatedPexBinaryInterpreterConstraints,
+        DeprecatedPexBinarySources,
```

Comment:
It's not clear that the field classes needed to be renamed when they were deprecated, but seems harmless.

Reply:
Agreed, they don't need to be. More to signal intent, e.g. make deletion of the deprecation simpler.